libinitializer/LedgerInitializer.cpp

```diff
@@ -92,19 +96,61 @@ void LedgerInitializer::initConfig(boost::property_tree::ptree const& _pt)
                      << LOG_KV("EINFO", boost::diagnostic_information(e)) << endl;
         BOOST_THROW_EXCEPTION(e);
     }
-    /// stop the node if there is no group
-    if (m_ledgerManager->getGroupList().size() == 0)
+    return newGroupIDList;
+}
+
+vector<dev::GROUP_ID> LedgerInitializer::foreachLedgerConfigure(
+    const string& _groupConfigPath, function<bool(dev::GROUP_ID const&, const string&)> _f)
```

Comment:
```suggestion
    const string& _groupConfigPath, function<bool(dev::GROUP_ID const&, const string&)> _initLedger)
```
I think we can make param more meaningful